### PR TITLE
Fixed source key name for docugami loader

### PR DESCRIPTION
**Pull request by tjaffri**
The Docugami loader was not returning the source metadata key. This was triggering this exception when used with retrievers, per https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/schema/prompt_template.py#L193C1-L195C41

The fix is simple and just updates the metadata key name for the document each chunk is sourced from, from "name" to "source" as expected.

I tested by running the python notebook that has an end to end scenario in it.

Tagging DataLoader maintainers @rlancemartin @eyurtsev

### CODE REVIEW

**Comment by tjaffri** [comment]
@rlancemartin @eyurtsev gentle ping on this please... thank you

**Comment by tjaffri** [comment]
Hi @rlancemartin @eyurtsev this should hopefully be a quick one to review and merge... thank you!

File: docs/extras/integrations/document_loaders/docugami.ipynb
Comment (baskaryan) [inline docs/extras/integrations/document_loaders/docugami.ipynb:1]: probably don't want to assume average use is using poetry

File: libs/langchain/langchain/document_loaders/docugami.py [hunk 1]
```diff
@@ -145,8 +146,8 @@ def _create_doc(node: Any, text: str) -> Document:
             """Create a Document from a node and text."""
             metadata = {
                 XPATH_KEY: _xpath_for_chunk(node),
-                DOCUMENT_ID_KEY: document["id"],
-                DOCUMENT_NAME_KEY: document["name"],
+                DOCUMENT_ID_KEY: document[DOCUMENT_ID_KEY],
+                DOCUMENT_SOURCE_KEY: document[DOCUMENT_NAME_KEY],
```
Comment (baskaryan) [inline libs/langchain/langchain/document_loaders/docugami.py:150]: technically a breaking change – maybe we include both "name" and "source"?

File: docs/extras/integrations/document_loaders/docugami.ipynb
Comment (hinthornw) [reply docs/extras/integrations/document_loaders/docugami.ipynb:1]: We could also start using `--quiet` to avoid having all these messages in the docs

Comment (tjaffri) [reply docs/extras/integrations/document_loaders/docugami.ipynb:1]: @baskaryan, thanks... the poetry install is there from before, but to address your feedback I updated the comment for users to select the appropriate install command for their usage

@hinthornw, thanks... I added --quiet flag.

File: libs/langchain/langchain/document_loaders/docugami.py (hunk 1 above)
Comment (tjaffri) [reply libs/langchain/langchain/document_loaders/docugami.py:151]: Done, thanks!

**Comment by tjaffri** [comment]
Feedback addressed, thank you @baskaryan @hinthornw

**Comment by baskaryan** [comment]
thanks @tjaffri!

**Pull request merged**